docs/framework/winforms/windows-forms-accessibility-improvements.md

```diff
@@ -0,0 +1,75 @@
+
+## Improved CheckedListBox Accessibility support
+On .NET Core, Windows Forms includes the following changes that affect the <xref:System.Windows.Forms.CheckedListBox control>:
+- Corrected CheckedListBox Bounds provided by accessibility properties for entries. Improved overall ListBox and CheckedListBox accessibility (corrected pproperty values and event model).
+
+## Improved ComboBox Accessibility support
+On .NET Core, Windows Forms includes the following changes that affect the <xref:System.Windows.Forms.ComboBox control>:
+- Updated the process of getting ComboBox items' accessibility objects to enable generating IDs for items instead of getting hash codes from items which may be unsafe in case of GetHashCode function is overriden.
+
+## Improved DataGridView Accessibility support
+On .NET Core, Windows Forms includes the following changes that affect the <xref:System.Windows.Forms.DataGridView control>:
+- Corrected DataGridView Bounds provided by accessibility properties for columns, rows, cells and corresponding headers, improved performance of bounding rectangle calculation. For now all accessibility bounds are represented correctly taking into account the bounds of entire control, its viewport and scrolls.
+- Corrected Value.IsReadOnly property value providing for accessible client applications. The property now shows correct IsReadOnly status for cells.
+- Fixed the issue with CellParsing event raising for the very first cell change. Cell value is now can be changed without any issues including the very first DataGridView control interaction.
+- Improved DataGridView background color contrast when using Windows High Contrast themes. Changed DataGridView default back color when using HC#1, HC#2, and HC Black themes.
+
+## Improved PropertyGrid Accessibility support
+On .NET Core, Windows Forms includes the following changes that affect the <xref:System.Windows.Forms.PropertyGrid control>:
+- Corrected PropertyGrid Bounds provided by accessibility properties for grid entries, improved performance of bounding rectangle calculation. For now all accessibility bounds are represented correctly taking into account the bounds of entire control, its viewport and scrolls.
+- Corrected accessible names and descriptions of subcontrols to not include control type names and to avoid double announcement for control type names.
+
+## Improved ToolStrip Accessibility support
+On .NET Core, Windows Forms includes the following changes that affect the <xref:System.Windows.Forms.ToolStrip control>:
+- Improved navigation through ToolStrip items. Corrected ToolStrip and Menu shift-tab navigation (back-looping the menu items when shift-tab up-arrow is pressed - navigate to the bottom menu element)
+- Improved Menu accessible navigation, corrected menu accessible control types for submenus to make submenus of type 'Menu' instead of 'MenuItem'.
```

Comment:
We have removed `Menu` and `MenuItem` types in 3.1. 
Please verify this sentence is still applicable

Reply:
What is there in its place?

Reply:
At this moment there are only ToolStrip, MenuStrip and StatusStrip controls. These controls fully cover the menu bar/tool bar functionality and there is no changes in the UI.

Reply:
Fixed @RussKie 's review point. Thanks!